src/HealthcareApis/HealthcareApis/help/Remove-AzHealthcareApisService.md

```diff
@@ -0,0 +1,213 @@
+schema: 2.0.0
+---
+
+# Remove-AzHealthcareApisService
+
+## SYNOPSIS
+Deletes a service instance.
+
+## SYNTAX
+
+### ServiceNameParameterSet (Default)
+```
+Remove-AzHealthcareApisService -Name <String> -ResourceGroupName <String> [-AsJob] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Remove-AzHealthcareApisService -InputObject <PSHealthcareApisService> [-AsJob] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Remove-AzHealthcareApisService [-ResourceId] <String> [-AsJob] [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Deletes a service instance.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-AzHealthcareApisService -Name MyService -ResourceGroupName MyResourceGroup
+```
+
+Deletes the existing HealthcareApis service with the provided name within a provided resource group.
+
+### Example 2
+```powershell
+PS C:\> $ResourceId = "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup/providers/Microsoft.HealthcareApis/services/MyService
+PS C:\> Remove-AzHealthcareApisService -ResourceId $ResourceId
+```
+
+Deletes the existing HealthcareApis service with the provided ResourceId.
+
+### Example 3
+```powershell
+PS C:\> Get-AzHealthcareApisService -ResourceGroupName MyResourceGroup -Name MyService | Remove-AzHealthcareApisService
+```
+
+Deletes the provided HealthcareApis service object.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet as a job in the background.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+HealthcareApis service object
+
+```yaml
+Type: Microsoft.Azure.Commands.HealthcareApis.Models.PSHealthcareApisService
+Parameter Sets: InputObjectParameterSet
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Name
+HealthcareApis Service Name.
+
+```yaml
+Type: System.String
+Parameter Sets: ServiceNameParameterSet
+Aliases: HealthcareApisName, FhirServiceName
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+{{ Fill PassThru Description }}
```

Comment:
@poadhika would you mind changing this line to "If provided, returns true if the operation was successful"?